apps/website/src/pages/docs/avatar.mdx

```diff
@@ -17,81 +17,59 @@ import * as AllExamples from 'examples';
   <AllExamples.AvatarMainExample client:load />
 </LiveExample>
 
-The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials. A user’s photo helps put a face to the name and makes it more personal. Any time an app requires a user profile, an avatar should be included. It also makes it very obvious to the user that they are logged in, as the non-logged in state displays differently.
+The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials.
 
-## Appearance
+## Usage
 
-### Sizes
-
-There are four sizes of avatars:
-
-<LiveExample src='Avatar.sizes.tsx'>
-  <AllExamples.AvatarSizesExample client:load />
-</LiveExample>
-
-- `small`
-- `medium`
-- `large`
-- `x-large`
-
-Which size to use is at your discretion. Determine which size is best for your context based on available space.
-
-### Initials
+Different properties can be passed to the `<Avatar>` component to customize it according to the user's needs.
```

Comment:
prefer using the term "props" (react concept) rather than "properties" (general concept)

```suggestion
Different props can be passed to the `<Avatar>` component to customize it according to the user's needs.
```